core/client/src/test/java/alluxio/hadoop/ConfUtilsTest.java

```diff
@@ -49,19 +47,17 @@ public void mergeEmptyHadoopConfigurationTest() {
   public void mergeHadoopConfigurationTest() {
     Configuration.defaultInit();
     org.apache.hadoop.conf.Configuration hadoopConfig = new org.apache.hadoop.conf.Configuration();
-    hadoopConfig.set(PropertyKey.S3N_ACCESS_KEY, TEST_S3_ACCCES_KEY);
-    hadoopConfig.set(PropertyKey.S3N_SECRET_KEY, TEST_S3_SECRET_KEY);
-    hadoopConfig.set(TEST_ALLUXIO_PROPERTY, TEST_ALLUXIO_VALUE);
```

Comment:
actually it is a good point. I have added this back and expect this non-Alluxio supported property not picked by Alluxio